resources/views/service/index_registration.blade.php

```diff
@@ -31,7 +31,7 @@
                 @foreach ($registrations as $registration)
                     @if ($registration->family)
                         <tr class="{{ $registration->family->leaving_on ? 'inactive' : 'active' }}">
-                            <td>{{ $registration->family->carers->first()->name }}</td>
+                            <td class="pri_carer">{{ $registration->family->carers->first()->name }}</td>
```

Comment:
Does it matter if we explode when there is no family or first carer found? I guess we expect there will always be both - but might want to wrap.